Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'activesupport', '~> 4.2'
+gem 'activesupport', '~> 5.0'
```

Comment:
At first, I tried to directly bump up to ActiveSupport 6. But Lokka currently supports Ruby 2.4 and AS 6 is not compatible with Ruby 2.4. As Ruby 2.4 is to sunset on the end of March 2020, we can bump up to AS 6 after April.